src/ProDOS.s

```diff
@@ -108,16 +109,16 @@ READ:       JSR   GETBLOCK    ; calc block address
             BNE   @ERROR      ; check for error
 
 @GETTOK:    LDA   #DUMMY      ; get data token
-            STA   DATA,X
+            JSR   WRDATA
```

Comment:
see comment on COMMAND

Reply:
This is also sent at low freq clock in my version